R/install.R

```diff
@@ -721,7 +721,7 @@ renv_install_preflight_requirements <- function(records) {
   fmt <- "Package '%s' requires '%s', but '%s' will be installed"
   text <- sprintf(fmt, format(package), format(requires), format(actual))
   if (renv_verbose()) {
-    renv_pretty_print(
+    notice(
```

Comment:
This is announcing an installation problem; should it be a caution not guarded by verbose?